### DISCUSSION TITLE
Open new file and set the current active connection

### DISCUSSION THREAD

**Discussion by svantevit** [dbeaver/dbeaver#22064] [Q&A]
Postgres database... let's say you have a file open, and active connection set (active connection & active catalog/schema). Then you open file (via File -> Open File), but the active connection for this file is not set. This is particularly annoying if you open multiple files, and instead of just executing a query against the active connection that you already set in your first file, you have to set connections to all files.

Is there an option to mark the current active connection as default, so all new open files would use the same one?

**Comment by LonwoLonwo** [comment]
Hello @svantevit 

Are you talking about a .sql file?

**Reply by ceuser1** [reply]
Hi @LonwoLonwo , yes I am talking about *.sql files.
